src/Simplex/Chat/Store/SQLite/Migrations/M20250217_superpeers.hs

```diff
@@ -0,0 +1,42 @@
+{-# LANGUAGE QuasiQuotes #-}
+
+module Simplex.Chat.Store.SQLite.Migrations.M20250217_superpeers where
+
+import Database.SQLite.Simple (Query)
+import Database.SQLite.Simple.QQ (sql)
+
+m20250217_superpeers :: Query
+m20250217_superpeers =
+  [sql|
+CREATE TABLE superpeers(
+  super_peer_id INTEGER PRIMARY KEY,
+  address TEXT NOT NULL,
+  name TEXT NOT NULL, -- domain to match with operators to be part of name/address, or separate field?
```

Comment:
should we save domains as separate field?

Reply:
you mean extract? yes, probably

Reply:
Should they have profile too?

Reply:
not sure they need it as a preset relay, owner/members will receive profile in group

Reply:
> you mean extract? yes, probably

for servers we don't extract it - it's in host. depends on whether we want names with or without domains, e.g. superpeer1.simplex.im